data/helpdata.txt

```diff
@@ -1553,18 +1537,18 @@ Of course, the different escape sequences can be combined in the same sentence,
 [help_worklist_editor]
 name = _("Worklist Editor")
 text = _("\
-The worklist editor is used to edit both worklists for each city (from the city dialog) and global worklists. \
+The worklist editor is used to edit both worklists for each city (from the City Dialog) and global worklists. \
 Using this editor you can create lists specifying what to build in the turns to come.\
 "), _("\
-To add an item to the worklist, click the plus ( + ) icon on the Production tab in the city dialog. A list of \
-available items will appear. You can hover over them for some brief help and then click the desired item  to \
+To add an item to the worklist, click the plus ( + ) icon on the Production tab in the City Dialog. A list of \
+available items will appear. You can hover over them for some brief help and then click the desired item to \
 add to the worklist of projects to build.\
 "), _("\
-To remove an item from the worklist simply double-click the item to remove. Use the buttons below the \
+To remove an item from the worklist simply double-click the item to remove it. Use the buttons below the \
```

Comment:
Consider rephrasing to "You can remove an item from the worklist by double-clicking on it."